apps/chef_db/src/chef_sql.erl

```diff
@@ -114,15 +105,7 @@
 -include_lib("sqerl/include/sqerl.hrl").
 -include("../../include/chef_types.hrl").
 
--type delete_query() :: delete_cookbook_version_by_id |
-                        delete_user_by_id |
-                        delete_data_bag_by_id |
-                        delete_data_bag_item_by_id |
-                        delete_environment_by_id |
-                        delete_client_by_id |
-                        delete_node_by_id |
-                        delete_role_by_id |
-                        delete_sandbox_by_id.
+-type delete_query() :: atom().
```

Comment:
Addition: and reasonably enough, hardly anyone remembered/knew to add new delete query values  here - I think this list is untouched nearly since our original implementation.